bpm/bonita-api/bonita-common-api-impl/src/main/java/org/bonitasoft/engine/bpm/flownode/impl/internal/CallActivityDefinitionImpl.java

```diff
@@ -98,6 +104,15 @@ public void setCallableElementType(final CallableElementType callableElementType
         this.callableElementType = callableElementType;
     }
 
+    public void addProcessStartContractInput(String inputName, Expression value) {

```

Comment:
On call activity, we KNOW that it starts a process, so the naming can be precise. But later on, we may use the type ContractInput on something else than just Process start action.